portal/server/api.js

```diff
@@ -3,7 +3,7 @@ const FormData = require('form-data');
 
 require('dotenv').config();
 
-const portalApi = process.env.DEAL_API_URL;
+const portalApi = process.env.PORTAL_API_URL;
```

Comment:
this line can be combined with the one below

Reply:
Have re-factored.